pkg/agent/params.go

```diff
@@ -28,10 +28,10 @@ func getParameters(config *NodeBootstrappingConfiguration, generatorCode string,
 
 	// Identify Master distro
 	if properties.MasterProfile != nil {
-		addValue(parametersMap, "osImageOffer", cloudSpecConfig.OSImageConfig[properties.MasterProfile.Distro].ImageOffer)
-		addValue(parametersMap, "osImageSKU", cloudSpecConfig.OSImageConfig[properties.MasterProfile.Distro].ImageSku)
-		addValue(parametersMap, "osImagePublisher", cloudSpecConfig.OSImageConfig[properties.MasterProfile.Distro].ImagePublisher)
-		addValue(parametersMap, "osImageVersion", cloudSpecConfig.OSImageConfig[properties.MasterProfile.Distro].ImageVersion)
+		addValue(parametersMap, "osImageOffer", cloudSpecConfig.OSImageConfig[api.Distro(properties.MasterProfile.Distro)].ImageOffer)
+		addValue(parametersMap, "osImageSKU", cloudSpecConfig.OSImageConfig[api.Distro(properties.MasterProfile.Distro)].ImageSku)
+		addValue(parametersMap, "osImagePublisher", cloudSpecConfig.OSImageConfig[api.Distro(properties.MasterProfile.Distro)].ImagePublisher)
+		addValue(parametersMap, "osImageVersion", cloudSpecConfig.OSImageConfig[api.Distro(properties.MasterProfile.Distro)].ImageVersion)
 		if properties.MasterProfile.ImageRef != nil {
 			addValue(parametersMap, "osImageName", properties.MasterProfile.ImageRef.Name)
 			addValue(parametersMap, "osImageResourceGroup", properties.MasterProfile.ImageRef.ResourceGroup)
```

Comment:
we can remove this if you want

Reply:
Removed in this PR: https://github.com/Azure/AgentBaker/pull/240